doc/reference/pipeline_spec.md

```diff
@@ -457,22 +490,25 @@ tuples of the datums in the inputs. For example:
 |        |        | (bar, buzz)     |
 ```
 
-Notice that cross inputs, do not take a name and maintain the names of the sub-inputs.
-In the above example you would see files under `/pfs/inputA/...` and `/pfs/inputB/...`.
+The cross inputs above do not take a name and maintain
+the names of the sub-inputs.
+In the example above, you would see files under `/pfs/inputA/...`
+and `/pfs/inputB/...`.
 
-`input.cross` is an array of inputs to cross, note that these need not be
-`pfs` inputs, they can also be `union` and `cross` inputs. Although there's no
-reason to take a cross of crosses since cross products are associative.
+`input.cross` is an array of inputs to cross.
+The inputs do not have to be `pfs` inputs. They can also be
+`union` and `cross` inputs. Although, there is
+ no reason to take a union of unions because union is associative.
 
 #### Cron Input
 
-Cron inputs allow you to trigger pipelines based on time. It's based on the
-unix utility `cron`. When you create a pipeline with one or more Cron Inputs
-pachd will create a repo for each of them. When a cron input triggers,
-that is when the present time satisfies its spec, pachd will commit
-a single file, named by the current [RFC
-3339 timestamp](https://www.ietf.org/rfc/rfc3339.txt) to the repo which contains the time which
-satisfied the spec.
+Cron inputs allow you to trigger pipelines based on time. A Cron input is
+based on the Unix utility called `cron`. When you create a pipeline with
+one or more Cron Inputs, `pachd` creates a repo for each of them. When
+a cron input triggers, that is when the present time satisfies its spec,
```

Comment:
> , that is when the present time satisfies its spec,

This phrase here is a bit kludgy IMO. Perhaps change:
, that is when the present time satisfies its spec, --> (e.g. when the present time satisfies its spec)